lib/tapioca/method_source_ext/code_helpers.rb

```diff
@@ -0,0 +1,38 @@
+# typed: true
+# frozen_string_literal: true
+
+module MethodSource
```

Comment:
We should rely on `yard-sorbet` instead: https://github.com/stripe/yard-sorbet/blob/master/lib/yard-sorbet/sig_handler.rb#L19